gateway/1.5/modules/ROOT/pages/policies-pdk-develop-custom-policies.adoc

```diff
@@ -2,4 +2,84 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+After installing all the xref:policies-pdk-prerequisites.adoc[], you can begin developing custom policies.
+
+The custom policy lifecycle consists of the following docs that explain all steps of developing a custom policy that occur in the Policy Development Kit (PDK). After you xref:policies-pdk-create-project.adoc[Create a New Project], you can complete the steps out of order, for example, when debugging a policy you may need to return to the configure policy feature or define a schema definition steps.
+
+The commands associated with each step are provided for reference. However, configuration is necessary for before running some of the commands.
+
+Complete the following steps to create a custom policy with PDK:
```

Comment:
Here you could soften this to something like "To get started creating a custom policy with PDK, here are the suggested steps:"